.github/workflows/hw-event-proxy.yaml

```diff
@@ -24,6 +24,7 @@ jobs:
           skip-build-cache: true
           # Required: the version of golangci-lint is required and must be specified without patch version: we always use the latest patch version.
           version: v1.41
+          working-directory: ./hw-event-proxy
```

Comment:
I think there are two workflow files with the same workflow.
hw-event-proxy.yaml & hw-event-proxy.yml need to delete one of them.. 

I think hw-event-proxy.yml is invalid because the flow steps seem to repeat after L39.

Reply:
The .yml files are somehow auto-generated. Removed.